pkg/vcs/git.go

```diff
@@ -672,3 +567,142 @@ func ParseGitDiff(patch []byte) []string {
 	}
 	return files
 }
+
+type GitWrapper struct {
```

Comment:
I would call this type `Git` and rename `git` to something else. The exported type is more important.